src/txn/txn_rollback_to_stable.c

```diff
@@ -1803,24 +1803,40 @@ __rollback_to_stable_btree_apply_all(WT_SESSION_IMPL *session, wt_timestamp_t ro
     struct timespec rollback_timer;
     WT_CURSOR *cursor;
     WT_DECL_RET;
-    uint64_t rollback_count, rollback_msg_count;
+    uint64_t max_count, rollback_count, rollback_msg_count;
     const char *config, *uri;
+    bool have_cursor;
 
     /* Initialize the verbose tracking timer. */
     __wt_epoch(session, &rollback_timer);
-    rollback_count = 0;
+    max_count = rollback_count = 0;
     rollback_msg_count = 0;
 
     WT_RET(__wt_metadata_cursor(session, &cursor));
+    have_cursor = true;
     while ((ret = cursor->next(cursor)) == 0) {
-        /* Log a progress message. */
-        __rollback_progress_msg(
-          session, rollback_timer, rollback_count, &rollback_msg_count, false);
-        ++rollback_count;
+        WT_ERR(cursor->get_key(cursor, &uri));
+        if (WT_BTREE_PREFIX(uri))
+            ++max_count;
+    }
+
+    WT_ERR_NOTFOUND_OK(ret, false);
+    WT_ERR(__wt_metadata_cursor_release(session, &cursor));
```

Comment:
Is there a reason we release the cursor and claim a new one rather than resetting?
This behaviour is the same as prior backports and develop so I assume it's correct.
CC @sueloverso